.travis.yml

```diff
@@ -10,23 +10,23 @@ python:
 env:
   - KARTOTHEK_ARROW_VERSION=0.12.1
   - KARTOTHEK_ARROW_VERSION=0.13.0
+  - KARTOTHEK_ARROW_VERSION=NIGHTLY
+  - KARTOTHEK_NP_PD_NIGHTLY=1
 
 before_install:
-  # https://github.com/JDASoftwareGroup/kartothek/issues/94
   - pip install --upgrade pip==19.1.*
   - pip install pip-tools
-  - ./compile_requirements.sh
+  - travis_wait 8 ./compile_requirements.sh
```

Comment:
The default timeout for travis commands is at 10min, i.e. 10min w/out any output will kill the job but I don't see what would take 10min in the compile requirements step. What's takes so much time here?